src/openforms/scss/components/builder/_builder.scss

```diff
@@ -112,3 +112,13 @@ div.flatpickr-calendar.open
     margin-right: 0;
   }
 }
+
+.gu-transit {
+  // #748 client requested increased contrast for this element
+  // here we clobber formio's compiled 0.2 opacity of the drag-and-drop target indicator
+  opacity: 0.7;
```

Comment:
FYI - the `-ms-filter` and `filter` rules are only needed for IE 8 which has long since been dead.